lib/simulation/openfisca/mapping/index.js

```diff
@@ -152,9 +164,18 @@ var buildOpenFiscaEntity = function(mesAidesEntity, mappingSchema, situation) {
             if ('default' in params) result[openfiscaKey] = params.default;
             else delete result[openfiscaKey];
         }
+
+        // On étend la valeur aux 3 derniers mois, succeptibles d'être appelés par le RSA et la PPA dans Openfisca
+        var value = result[openfiscaKey];
+        if (! _.isUndefined(value) && ! params.doNotExtend) {
+            result[openfiscaKey] = {};
+            monthKeys.forEach(function(monthKey) {
```

Comment:
Il me semblerait plus lisible d'itérer directement sur les clés de mois ici plutôt que de passer par une variable intermédiaire `monthKeys`.